frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -148,6 +148,16 @@ describe('Kubernetes resource CRUD operations', () => {
         expect(crudView.resourceTitle.getText()).toEqual(name);
       });
 
+      it('event view displays created resource instance', async () => {
```

Comment:
Hey @zherman0 I'm confused by this test change. This doesn't look right to me. Why is this change needed?

Reply:
I was trying to add an event page specific test, but that proves to be the wrong place.  I will clean up that file shortly.

Reply:
OK, I see. I didn't realize we had this test gap. Thanks for adding something 👍 

It should be outside this loop or it will re-run the tests for every resource.

Reply:
I purposely was trying to leave it in that loop, but it turns out that not all the resources show on the events page, which is why my test was failing.  I have created a specific block for it to run in and I will just test a few resources that I know will show up.